modules/ocf/manifests/firewall/post.pp

```diff
@@ -0,0 +1,71 @@
+class ocf::firewall::post {
+  # Only allow root and postfix to connect to anthrax port 25; everyone else
+  # must use the sendmail interface
+  ['root', 'postfix'].each |$username| {
+    ocf::firewall::firewall46 {
+      "996 allow ${username} to send on SMTP port":
+        opts => {
+          chain  => 'PUPPET-OUTPUT',
+          proto  => 'tcp',
+          dport  => 'smtp',
+          uid    => $username,
+          action => 'accept',
+        },
+        before => undef,
+    }
+  }
+
+  ocf::firewall::firewall46 {
+    '997 forbid other users from sending on SMTP port':
+      opts => {
+        chain  => 'PUPPET-OUTPUT',
+        proto  => 'tcp',
+        dport  => 'smtp',
+        action => 'drop',
+      },
+      before => undef,
+  }
+
+
+  # Special devices we want to protect from most hosts
+  $devices = ['corruption-mgmt','hal-mgmt', 'jaws-mgmt', 'pagefault', 'pandemic-mgmt',
```

Comment:
Maybe move these arrays down closer to where they are used so that it is clearer what they are doing?